Util.hs

```diff
@@ -149,6 +172,10 @@ run name tool opts = do
             copyFile (name ++ "-fbuild.py") "fbuildroot.py"
             system_ $ "fbuild -j" ++ show p ++ " " ++ target ++ " > " ++ devNull
             removeFile "fbuildroot.py"
+        Gup -> do
+            createSymbolicLink (name ++ "-gup") "gup"
```

Comment:
What about a recursive file copy, with a `writeFileChanged` function that only overwrites a file if it is different? Would that work? (If the Haskell required to do that is going to be tricky for you to formulate, leave it as `createSymbolicLink` and I'll do that later).

If "pool" must be non-Windows, I can mark it as such for just gup (no current way to do that, but I'm sure it's possible to add). I'll try installing GNU Parallel first though, which may work with Cygwin.